src/components/App/App.scss

```diff
@@ -0,0 +1,18 @@
+body {
```

Comment:
Why do you not use scss variables?

Reply:
How about adding `reset.css` or `normalize.css` to the project?

Reply:
It's too small project, I think those files would be redundant. Moreover I don't need to reset all default styles.